docs/DeveloperGuide.md

```diff
@@ -2,37 +2,189 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
 
 ## Design & implementation
```

Comment:
Should this header Design & implementation be removed, since there is a separate Implementation header below?